WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShippingCreateLabelsView.swift

```diff
@@ -22,6 +22,19 @@ struct WooShippingCreateLabelsView: View {
 
     @Environment(\.dismiss) private var dismiss
 
+    /// Whether the shipment details bottom sheet is expanded.
+    @State private var isShipmentDetailsExpanded = false
+
+    // TODO: 14044 - Track the paper size selection in the view model
+    // TODO: 13558 - Set the default selection from the account settings
+    @State private var selectedPaperSize: String = "Label (4\"x6\")"
+
+    // TODO: 14044 - Replace this static list with real data from the store
+    private let paperSizes: [String] = [
+        "Letter (8.5\"x11\")",
+        "Label (4\"x6\")"
+    ]
```

Comment:
We should be able to reuse `ShippingLabelPaperSize` for this, although it looks like in the legacy flow we always show the same options (defined in the app rather than getting a list of options from the backend). We should double check whether the store has a defined list of options we can get from the API.